src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -455,3 +466,15 @@ COORD Terminal::_ConvertToBufferCell(const COORD viewportPos) const
     THROW_IF_FAILED(ShortAdd(positionWithOffsets.Y, gsl::narrow<SHORT>(_ViewStartIndex()), &positionWithOffsets.Y));
     return positionWithOffsets;
 }
+
+// Method Description:
+// - This method won't be used. We just throw and do nothing. For now we
+//   need this method to implement UiaData interface
+// Arguments:
+// - coordSelectionStart - Not used
+// - coordSelectionEnd - Not used
+// - attr - Not used.
+void Terminal::ColorSelection(const COORD, const COORD, const TextAttribute)
+{
+    THROW_HR(E_NOTIMPL);
+}

```

Comment:
Just attach the work item Michael was talking about here and I'll approve it.

Reply:
https://github.com/microsoft/terminal/issues/3542